pkg/cli/aws/aws_test.go

```diff
@@ -161,17 +167,54 @@ func Test_refreshCredsViaOktaAWSCLI(t *testing.T) {
 		copts := DefaultCredentialOptions()
 		copts.Log = log
 
+		b := &box.Config{}
+
 		acopts := &AuthorizeCredentialsOptions{
 			DryRun: true,
 			Output: OutputCredentialProvider,
 		}
 
-		err := refreshCredsViaOktaAWSCLI(context.Background(), copts, acopts, "")
+		err := refreshCredsViaOktaAWSCLI(context.Background(), copts, acopts, b, "")
 		assert.NilError(t, err)
-		assert.Equal(t, len(hook.Entries), 2)
+		expectedEntryCount := 2
+		if os.Getenv("CI") != "" {
+			// extra log for missing okta-aws-cli since it's not installed by CI
+			expectedEntryCount = 3
+		}
+		assert.Equal(t, len(hook.Entries), expectedEntryCount)
 		msg := hook.LastEntry().Message
 		assert.Assert(t, strings.HasPrefix(msg, "Dry Run: okta-aws-cli"))
 		assert.Assert(t, !strings.Contains(msg, "--write-aws-credentials"))
-		assert.Assert(t, strings.Contains(msg, "--format credential-provider"))
+		assert.Assert(t, cmp.Contains(msg, "--format process-credentials"))
 	})
 }
+
+func Test_oktaAwsCliVersionOutputMatchesV1(t *testing.T) {
+	t.Run("version 1 matches", func(t *testing.T) {
```

Comment:
nit: If you're going to use sub-tests, it would be preferable to use [table tests](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests)

Reply:
Replaced with non-sub-tests